playwright.go

```diff
@@ -93,6 +93,48 @@ func (p *Playwright) WaitForSelector(selector string, opts playwright.PageWaitFo
 	}
 }
 
+func (p *Playwright) WaitForURL(url string, opts playwright.FrameWaitForURLOptions) {
```

Comment:
We may want to call out that this only works with exact URLs -- a departure from the JS api that works with regex
```suggestion
// URL string must be the exact URL unlike the regex/pattern suppor of the JS library
func (p *Playwright) WaitForURL(url string, opts playwright.FrameWaitForURLOptions) {
```